src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -69,6 +88,23 @@ case class AgoraEntity(@(ApiModelProperty@field)(required = true, value = "The n
                        payload: Option[String] = None
                         )
 
+@ApiModel(value = "Agora Search")
```

Comment:
Is there any reason we can't just always return a Seq of AgoraEntities? In which case we'd only need one marshaller/unmarshaller.